packages/orbit-components/src/Collapse/Collapse.stories.tsx

```diff
@@ -30,21 +56,17 @@ export const Default = ({ label }) => {
         maxValue={24}
       />
     </Collapse>
-  );
-};
+  ),
 
-Default.story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["initialExpanded", "expanded"],
```

Comment:
Is there a reason why these are excluded in all the stories? Can't we have them in controls? There is no playground 🤷